molgenis-security-ui/src/main/frontend/package.json

```diff
@@ -13,9 +13,16 @@
     "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs",
     "build": "node build/build.js"
   },
+  "peerDependencies": {
+    "bootstrap": "4.0.x",
+    "jquery": "1.9.x"
```

Comment:
It seems strange to have a peerDependency on jquery 1.9 and a dependency on 3.3 , and why do we even need jquery ? could we remove these or are these needed for bootstrap-vue ( i do not think it  is needed , when looking a the docs)

Same for bootstrap dependency